Runtime/CesiumIonServer.cs

```diff
@@ -0,0 +1,125 @@
+        /// is embedded in packaged games for use at runtime.
+        /// </summary>
+        public string defaultIonAccessToken;
+
+        /// <summary>
+        /// Gets the default Cesium ion Server (ion.cesium.com).
+        /// </summary>
+        /// <remarks>
+        /// It is expected to be found at `Assets/CesiumSettings/CesiumIonServers/ion.cesium.com`.
+        /// In the Editor, it will be created if it does not already exist, so this method always
+        /// returns a valid instance. At runtime, this method returns null if the object does not
+        /// exist.
+        /// </remarks>
+        public static CesiumIonServer defaultServer
+        {
+            get
+            {
+                CesiumIonServer result = Resources.Load<CesiumIonServer>("CesiumIonServers/ion.cesium.com");
+
+#if UNITY_EDITOR
+                if (result == null)
+                {
+                    if (!AssetDatabase.IsValidFolder("Assets/CesiumSettings"))
+                        AssetDatabase.CreateFolder("Assets", "CesiumSettings");
+                    if (!AssetDatabase.IsValidFolder("Assets/CesiumSettings/Resources"))
+                        AssetDatabase.CreateFolder("Assets/CesiumSettings", "Resources");
+                    if (!AssetDatabase.IsValidFolder("Assets/CesiumSettings/Resources/CesiumIonServers"))
+                        AssetDatabase.CreateFolder("Assets/CesiumSettings/Resources", "CesiumIonServers");
+                    result = ScriptableObject.CreateInstance<CesiumIonServer>();
+
+#pragma warning disable 618
+                    if (!string.IsNullOrEmpty(CesiumRuntimeSettings.defaultIonAccessTokenID))
+                    {
+                        if (string.IsNullOrEmpty(result.defaultIonAccessTokenId))
+                        {
+                            result.defaultIonAccessTokenId = CesiumRuntimeSettings.defaultIonAccessTokenID;
+                        }
+                        CesiumRuntimeSettings.defaultIonAccessTokenID = "";
+                    }
+
+                    if (!string.IsNullOrEmpty(CesiumRuntimeSettings.defaultIonAccessToken))
+                    {
+                        if (string.IsNullOrEmpty(result.defaultIonAccessToken))
+                        {
+                            result.defaultIonAccessToken = CesiumRuntimeSettings.defaultIonAccessToken;
+                        }
+                        CesiumRuntimeSettings.defaultIonAccessToken = "";
+                    }
+
+#pragma warning restore 618
+
+                    AssetDatabase.CreateAsset(result, "Assets/CesiumSettings/Resources/CesiumIonServers/ion.cesium.com.asset");
+                    AssetDatabase.Refresh();
+                }
+#endif
+
+                return result;
+            }
+        }
+
+        /// <summary>
+        /// Gets the current Cesium ion server that should be assigned to newly-created objects.
+        /// </summary>
+        public static CesiumIonServer currentForNewObjects
```

Comment:
For consistency with the Unreal PR

```suggestion
        public static CesiumIonServer serverForNewObjects
```